pkg/util/kubernetes/kubelet/types_kubelet.go

```diff
@@ -57,6 +57,7 @@ type Spec struct {
 	SecurityContext   *PodSecurityContextSpec `json:"securityContext,omitempty"`
 	RuntimeClassName  *string                 `json:"runtimeClassName,omitempty"`
 	Tolerations       []Toleration            `json:"tolerations,omitempty"`
+	RestartPolicy     string                  `json:"restartPolicy,omitempty"`
```

Comment:
Is it used anywhere, as the `container.restarts` metric does not make sense on Kubernetes.